endpoints/remotehost/remotehost

```diff
@@ -193,7 +193,7 @@ function process_remotehost_opts() {
                 ;;
             osruntime)
                 case "$val" in
-                    builtin|chroot|podman)
```

Comment:
and where is the actual code for this option ? not implemented ?

Reply:
pretty much, it's the same as some of the other cruft I have been removing...this was used early in the bootstrapping of Crucible/rickshaw and is just totally stale/dead at this point...I couldn't find any code to go along with the option